lib/lotus/action/cookie_jar.rb

```diff
@@ -131,6 +137,7 @@ def extract(env)
         hash   = env[COOKIE_HASH_KEY] ||= {}
         string = env[HTTP_HEADER]
 
+        return if string && string.include?(RACK_SESSION_KEY)
```

Comment:
there isn't communication between sessions middleware and cookies jar (now), I'll try to inspect in the bottom of this method how the cookie is converted with `Rack::Utils`